docs/build_arm.md

```diff
@@ -31,20 +32,28 @@ use the optimized 64-bit kernels of LCE on a Raspberry Pi, a 64-bit OS such as
 First configure Bazel using the instructions [here](build.md). Make sure you
 have run the `./configure.sh` script as instructed.
 
-To cross-compile an LCE inference binary for ARM architectures, the bazel
+To cross-compile the LCE example application for ARM architectures, the bazel
 target needs to be built with the `--config=rpi3` (32-bit ARM) or
-`--config=aarch64` (64-bit ARM) flag. For example, to build the LCE benchmark
-tool, run the following command from the LCE root directory:
+`--config=aarch64` (64-bit ARM) flag. For example, to build the example
```

Comment:
why do we need to have two different examples here? The example itself is not what matters, its about building and the building for both examples, beside just one flag, is the same.

Reply:
True. I'll just leave the `lce_minimal` example and then in the text state the name of the benchmark bazel target.